appcues/src/main/java/com/appcues/ui/composables/CompositionRemembers.kt

```diff
@@ -49,7 +50,9 @@ internal fun LaunchOnHideAnimationCompleted(block: () -> Unit) {
     with(remember { mutableStateOf(isContentVisible) }.value) {
         // if hide animation is completed
         if (isIdle && currentState.not()) {
-            block()
+            SideEffect {
+                block()
+            }
```

Comment:
testing showed that this call to `dismiss()` that happens during hide can throw an exception, if the underlying Android View that is presenting our content (ViewPresenter) is removed while the content is composing (inside a Composable). I can no longer find the resource I had, but basically, if you are removing the ComposeView out from under a Composition, it needs to happen in a SideEffect so it runs after the Composition completes.